draft-ietf-rats-architecture.md

```diff
@@ -372,13 +261,105 @@ The FIDO protocol supports several remote attestation protocols and a mechanism
+
+# Terminology {#terminology}
+
+## Roles
+
 Attester:
 
-: Every FIDO Authenticator contains an Attester.
+: A role performed by an entity (typically a device) whose Evidence must be appraised in order to infer the extent to which the Attester is considered trustworthy, such as when deciding whether it is authorized to perform some operation.
+Produces: Evidence.
+
 
 Relying Party:
 
-: Any web site, mobile application back end or service that does biometric authentication.
+: A role performed by an entity that depends on the validity of information about an Attester, for purposes of reliably applying application specific actions.  Compare /relying party/ in {{RFC4949}}
+Negotiates: the relationship between an Attester and a Verifier.
+Consumes: Attestation Results
+
+
+Verifier:
+
+: A role performed by an entity that appraises the validity of Evidence about an Attester
+ and produces Attestation Results to be used by a Relying Party.
+Consumes: Evidence, Reference Values, Endorsements, Appraisal Policy for Evidence;
+Produces: Attestation Results.
+
+
+Relying Party Owner:
+
+: An entity (typically an administrator), that is authorized to configure Appraisal Policy for Attestation Results in a Relying Party.
+Produces: Appraisal Policy for Attestation Results.
+
+
+Verifier Owner:
+
+: An entity (typically an administrator), that is authorized to configure Appraisal Policy for Evidence in a Verifier.
+Produces: Appraisal Policy for Evidence.
+
+
+Endorser:
+
+: An entity (typically a manufacturer) whose Endorsements help Verifiers appraise the authenticity of Evidence.
+Produces: Endorsements.
+
+Reference Value Provider:
+: An entity (typically a manufacturer) whose Reference Values help Verifiers appraise the authenticity of Evidence.
+Produces: Reference Values.
+
+
+## Artifacts
+
+Claim:
+: A piece of asserted information, often in the form of a name/value pair. Claims make up the usual structure of Evidence.
+(Compare /claim/ in {{RFC7519}})
+
+Evidence:
+
+: A set of information about an Attester that is to be appraised by a Verifier.
+Evidence may include configuration data, measurements, telemetry, or inferences.
+Used By: Verifier;
+Produced By: Attester
+
+
+Attestation Result:
+
+: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results.
+Used By: Relying Party;
+Produced By: Verifier.
+
+
+Appraisal Policy for Evidence:
+
+: A set of rules that informs how a Verifier evaluates the validity of information about an Attester. Compare /security policy/ in {{RFC4949}}.
+Used by: Verifier;
+Produced by: Verifier Owner.
+
+
+Appraisal Policy for Attestation Results:
+
+: A set of rules that direct how a Relying Party uses the Attestation Results regarding an Attester generated by the Verifiers. Compare /security policy/ in {{RFC4949}}.
+Used by: Relying Party;
+Produced by: Relying Party Owner.
+
```

Comment:
Another problem is that this PR deletes the definition of Endorsements from line 178 and doesn't add it back into this terminology section.